pkg/core/storage/redis_store.go

```diff
@@ -6,6 +6,14 @@ import (
 	"github.com/go-redis/redis"
 )
 
+// RedisDBOptions configuration for RedisDB.
+type RedisDBOptions struct {
+	DataDirectoryPath string `yaml:"DataDirectoryPath"`
```

Comment:
`DataDirectoryPath` for Redis? And how is it used by Redis?

Reply:
not used, right